app/src/components/Search/Search.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
I recommend adding some tests to this component: entering text into the search field, checking if setSearchQuery was properly called